metrics/grafana/tikv_details.dashboard.py

```diff
@@ -1444,6 +1444,25 @@ def ThreadCPU() -> RowPanel:
             ),
         ]
     )
+    layout.row(
+        [
+            graph_panel(
+                title="IME CPU",
+                description="The CPU utilization of IME threads",
+                yaxes=yaxes(left_format=UNITS.PERCENT_UNIT),
+                targets=[
+                    target(
+                        expr=expr_sum_rate(
+                            "tikv_thread_cpu_seconds_total",
+                            label_selectors=['name=~"ime.*"'],
+                            by_labels=["instance", "name"],
+                        ),
+                        legend_format="{{instance}}-{{name}}",
```

Comment:
Shall we only display a total cpu by default and hide the details cpu usage per-type?

Reply:
Since the dashboard named "tikv-details", let's keep the details.

Reply:
Based on the fact that IME will start quite a number of threads and most of them only consume very little cpu time in most time(if not all), I wonder what is the benefit of show these noise-like data by default.

Reply:
Done.